### ISSUE TITLE
1.8.10 - Universal Crafting Grid

### ISSUE THREAD

**Issue by Hannes1909** [AllTheMods/ATM-6#2711]
Hi all,

after crafting and using the universal crafting grid the first time, my normal fluid grids show "normal" items as well:
![grafik](https://user-images.githubusercontent.com/29476657/142722014-aca15507-9813-4373-8ae2-a173ecf2eafb.png)

The other way round, too - normal crafting grid or wireless crafting grid show fluids and seem to start counting items all over:
![grafik](https://user-images.githubusercontent.com/29476657/142722050-f13c23e7-0dc7-4111-85fa-3c7d32e5d925.png)

I imported my save from 1.8.9 (copied save folder from old instance) and crafted the universal crafting grid right away.

**Comment by Aavar6802** [comment]
I have the same issue.

**Comment by starforcraft** [comment]
Log? And are you on server?

**Comment by Aavar6802** [comment]
After it recounted all the items in refined storage it is now back to normal. It is almost like it had to remove and reinsert all the items in the disks.

**Comment by Hannes1909** [comment]
> It recounts all of the items in the refined storage system every time you use the universal grid.
> 
> This is on a server. The logs show nothing as this is happening.

Thanks for the hint - Ill try tomorrow, singleplayer / LAN-Server. Sadly I couldn't find anything either in the logs. If it is only a temporary issue, its totally fine with me letting Refined Storage recount everything once, but hopefully not every time, I just feared that this bug killed my savegame and reverted to 1.8.9 instantly.

**Comment by jllllll** [comment]
I am playing on 1.8.14 and had to delete the universal crafting grid almost immediately after using it. It caused my normal grids to multiply inserted item counts by over 20k and cause the storage system to slow down tremendously. I did not have fluid storage yet, so I did not observe the fluid interactions seen above, however, trying to insert patterns into crafters caused severe memory leaks forcing me to close the game. Fortunately, fully deleting the universal crafting grid and then restarting the game fixed my storage system.
